app/helpers/form_errors_builder.rb

```diff
@@ -0,0 +1,29 @@
+class FormErrorsBuilder < ActionView::Helpers::FormBuilder
```

Comment:
Why would we go for the alternative just because of slightly reduced complexity (so soon after last nights facade pattern discussion :wink:)?
- much of the difference is html noise between a list and a bootstrap panel
- we have encapsulated all of the form handling for the object in a form_for model why would we put parts of it elsewhere?